test/lib/y2users/useradd_config_test.rb

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env rspec
```

Comment:
Please, include this new file in _test/Makefile.am_

Reply:
Done